frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/SlideInBackgroundable.java

```diff
@@ -82,9 +81,6 @@ public void run(ProgressIndicator indicator) {
             .onFailure(EntryDoesNotExistException.class, exceptionWithMessageHandler(
                 format(getString("action.GitMachete.BaseSlideInBranchBelowAction.notification.message.entry-does-not-exist"),
                     entryToSlideIn.getName())))
-            .onFailure(EntryIsRootException.class, exceptionWithMessageHandler(
-                format(getString("action.GitMachete.BaseSlideInBranchBelowAction.notification.message.entry-is-root"),
-                    entryToSlideIn.getName())))
             .onFailure(Exception.class, exceptionWithMessageHandler(/* message */ null))
```

Comment:
Hmm theoretically this branch will never happen (coz `com.virtuslab.branchlayout.api.IBranchLayout#slideOut` only throws `EntryDoesNotExistException`)... maybe just use good ol' `try .. catch` here